packages/dashboard/rollup.config.js

```diff
@@ -40,6 +40,7 @@ export default [
       json(),
       typescript({
         tsconfig: './tsconfig.json',
+        exclude: ['**/*.test.ts', '**/*.test.tsx', '**/*.spec.ts', '**/*.spec.tsx'],
```

Comment:
it should be 
``` 
typescript({
  tsconfig: './tsconfig.json',
  tsconfigOverride: {
    exclude: ['**/*.test.ts', '**/*.test.tsx', '**/*.spec.ts', '**/*.spec.tsx'],
  },
}),
```

Reply:
https://github.com/awslabs/iot-app-kit/commit/49510bacc6f5c385cd3d153aa0a127b5b94dfc70#diff-8d2edfca7863606e1df4d8d8da1e7150063abfa13c9a2bfe654f9fbeacc1a60cL41-L43

Reply:
fixed :-)